sdk/formrecognizer/Azure.AI.FormRecognizer/src/FieldValueType.cs

```diff
@@ -10,55 +10,50 @@ namespace Azure.AI.FormRecognizer.Models
     /// <summary>
     /// The type of the value of a <see cref="FormField"/>.
     /// </summary>
-    [CodeGenModel("FieldValueType")]
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Naming", "CA1720:Identifier contains type name")]
```

Comment:
Do we not still need `[CodeGenModel("FieldValueType")]` here to associate this with the generated type?

Reply:
no, we only need to use the annotations when renaming something. 
see https://github.com/Azure/autorest.csharp#renaming-an-enum-member